templates/terraform/unordered_list_customize_diff.erb

```diff
@@ -0,0 +1,34 @@
+keys := diff.GetChangedKeysPrefix(<%= go_literal(Google::StringUtils.underscore(prop.name)) -%>)
+if len(keys) < 1 {
+	return nil
+}
+count := diff.Get("<%= Google::StringUtils.underscore(prop.name) -%>.#").(int)
+if count < 1 {
+	return nil
+}
+old := make([]interface{}, count)
+new := make([]interface{}, count)
+for i := 0; i < count; i++ {
+    o, n := diff.GetChange(fmt.Sprintf("<%= Google::StringUtils.underscore(prop.name) -%>.%d", i))
+
+    if o != nil {
+        old = append(old, o)
+    }
+    if n != nil {
+        new = append(new, n)
+    }
+}
+if len(old) != len(new) {
+    return nil
+}
+
+for o, _ := range old {
```

Comment:
This logic seems wrong to me. First off, o and n are the array indices. Second, this looks like it'll return nil (i.e. keep doing what you're doing) as soon as any two elements aren't equal. What if instead, we loaded everything into a schema.Set and then checked equality on that?

Reply:
Weird that the tests passed - yikes, thanks for catching that.

Loading into a schema.Set is a good plan, I'll try that.  I tried with regular go sets, but of course that didn't work since `[]interface{}` isn't hashable.

Reply:
That's much better, thank you.